test/autoschedulers/mullapudi2016/cost_function.cpp

```diff
@@ -48,7 +49,22 @@ int main(int argc, char **argv) {
     // Auto-schedule the pipeline
     Target target = get_jit_target_from_environment();
     Pipeline p(stencils[num_stencils - 1]);
-    AutoSchedulerResults results = p.apply_autoscheduler(target, {"Mullapudi2016"});
+
+    // Down-adjust the GPU's L2 cache size (aka shared memory) estimate from
+    // 48kB to 25kB, so as to workaround the following error:
+    //
+    // Error: CUDA error: CUDA_ERROR_INVALID_VALUE cuLaunchKernel failed
+    //
+    // GPU kernel launch failure happens when Likely because (i) most Halide
+    // users can only afford consumer-grade GPUs (e.g. Nvidia GTX1660), and (ii)
+    // the Autoscheduler heristics is over-confident about the actual L2/share
+    // memory usage of the generated GPU kernels.
+    constexpr unsigned last_level_cache_estimate_for_gpu = 25'000;
+    constexpr unsigned using_default_values = 0;
+
+    AutoSchedulerResults results = p.apply_autoscheduler(target,
+                                                         get_autoscheduler_params(target.has_gpu_feature(),
+                                                                                  target.has_gpu_feature() ? last_level_cache_estimate_for_gpu : using_default_values));
```

Comment:
The need of `last_level_cache_size` manual tuning is required by design. Related to #8637 .

cc'ed @mcourteaux

Reply:
> Can you also format the arguments on the next line, to avoid these extremely long lines? 

@alexreinking Manual code formatting requires manual override with `// clang-format off`. Is it frown upon by reviewers?

Reply:
I believe you can simply add a newline after an opening `(`, and clang-format will accept that and move stuff to the left.